main.tf

```diff
@@ -55,28 +54,28 @@ resource "ibm_resource_key" "cos_credentials" {
   role                 = local.role
   parameters           = { "HMAC" = true }
 
-  //User can increase timeouts
+  # User can increase timeouts
   timeouts {
     create = "15m"
     delete = "15m"
   }
 }
 
 module "clis" {
```

Comment:
It this really needed as opposed to indicate the dependencies to consumer (or providing a link on how to set up dependencies via a script).
There may be a few issues:
- Forcing download from external sources may be an issue for some customers running their terraform in limited connectivity / airgapped. For instance from a Schematics remote worker (Agent)
- Some customers may want to finely control the version of those dependencies - eg: to ensure use of dependency versions with no CVEs